packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -151,6 +154,14 @@ const OverflowMenu = React.forwardRef((props, ref) => {
       });
   };
 
+  const extractRenderConfirmation = () => {
```

Comment:
Not very elegant solution, however it should unblock us. I think, we should revisit this component and refactor the way we render confirmation modal. Extracting it from children - doesn't seem to be the best option? Perhaps we should render it from inside `OverflowMenu.Item` ? Also, the today's implementation doesn't support multiple items with Confirmation dialogs.